man/systemd-system.conf.xml

```diff
@@ -161,8 +161,9 @@
         <literal>d</literal>, <literal>w</literal>). If
         <varname>RuntimeWatchdogSec=</varname> is set to a non-zero
         value, the watchdog hardware
-        (<filename>/dev/watchdog</filename>) will be programmed to
-        automatically reboot the system if it is not contacted within
+        (<filename>/dev/watchdog</filename> or the path to which
```

Comment:
"\<filename>/dev/watchdog\</filename>  or the path specified with \<varname>WatchdogDevice=\</varname> or the kernel option \<varname>systemd.watchdog-device=\</varname>"